### ISSUE TITLE
messages of reportSlack always uses default color

### ISSUE THREAD

**Issue by a16** [nttgin/BGPalerter#626]
**Describe the bug**
It doesn't work params.colors in reportSlack.

**Provide an example**
I'm using the following config.

```
reports:
  - file: reportSlack
    channels:
      - hijack
      - newprefix
      - visibility
      - path
      - misconfiguration
      - rpki
    params:
      showPaths: 0 # Amount of AS_PATHs to report in the alert
      colors:
        hijack: '#d60b1c'
        newprefix: '#fa9548'
        visibility: '#fad648'
        path: '#42cbf5'
        rpki: '#d892f0'
      hooks:
        default: **webhook url**
 ```
Then I got this. all events are blue(default color). 
![スクリーンショット 2021-08-12 16 38 08](https://user-images.githubusercontent.com/5094724/129157133-0c73b23d-035b-4484-8e5d-23e3c17d4569.jpg)

Am I wrong?

**Are you using the binary or the source code?**
I'm using binary code (v1.28.2)

**Comment by massimocandela** [comment]
Hi @a16,

This was reported and fixed in the PR: https://github.com/nttgin/BGPalerter/pull/622
It will go out with the next release. Thanks for reporting anyway!

**Comment by massimocandela** [comment]
released in v1.28.3
